scenes/menus/level_designer/follow_viewport_workaround.gd

```diff
@@ -2,8 +2,10 @@ extends CanvasLayer
 
 # As of godot 3.5, buttons inside of a canvaslayer with follow viewport enabled do not work
 # So this is basically a script which makes the canvaslayer follow the viewport with it still working
+# TODO: Game is on 4.1 now. Check if this is still true.
```

Comment:
No longer applies. This script can be ditched, and Follow Viewport can be safely enabled.